cfme/services/requests.py

```diff
@@ -257,6 +261,8 @@ def find_request(self, cells, partial_check=False):
             partial_check: If to use the ``__contains`` operator
         Returns: row
         """
+        # TODO Remove after we'll have any other machanism to wait for
+        time.sleep(3)
```

Comment:
because table.attributized_headers was rushing too fast on 5.9. @mfalesni is it possible to handle this through widget itself? Table in particular

Reply:
Maybe you can use `wait_for` here. Like:
```python
wait_for(lambda: self.table.is_displayed)
```